amical/get_infos_obs.py

```diff
@@ -231,3 +231,79 @@ def get_pixel_size(ins):
     except KeyError:
         p = np.NaN
     return p
+
+
+def get_ifu_table(
+    i_wl, filtname="YH", instrument="SPHERE-IFS", verbose=False, display=False
+):
+    """Get spectral information for the given instrumental IFU setup.
+    `i_wl` can be an integer, a list of 2 integers (to get a range between those
+    two) or a list of integers (>= 3) used to display the
+    requested spectral channels."""
+    from matplotlib import pyplot as plt
+
+    wl = get_wavelength(instrument, filtname) * 1e6
+
+    if np.isnan(wl.any()):
```

Comment:
what ? doesn't `wl.any()` returns a boolean ? in what scenario could its return value be `np.nan` ?

Reply:
Indeed, now we deal the potential issue within get_wavelength(), so no need of this check anymore.